### PR TITLE
Add Remove Liquidity view

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

**Comment by lazarmitic** [comment]
Review and merge after https://github.com/tempus-finance/tempus-app/pull/464 has been merged.

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+              <Spacer size={15} />
+              <Typography variant="card-body-text">{estimatedPrincipalsFormatted}</Typography>
+            </div>
+          </SectionContainer>
+          <PlusIconContainer orientation="vertical" />
+          <SectionContainer>
+            <Typography variant="h4">Yields</Typography>
+            <Spacer size={10} />
+            <div className="tf__flex-row-center-v">
+              <Typography variant="card-body-text">Estimated</Typography>
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx:186]: use translations please

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+              <Spacer size={15} />
+              <Typography variant="card-body-text">{estimatedPrincipalsFormatted}</Typography>
+            </div>
+          </SectionContainer>
+          <PlusIconContainer orientation="vertical" />
+          <SectionContainer>
+            <Typography variant="h4">Yields</Typography>
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx:183]: use translations please

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        </SectionContainer>
+      </SectionContainer>
+      <Spacer size={15} />
+      <SectionContainer title="to">
+        <div className="tf__flex-row-center-v">
+          <SectionContainer>
+            <Typography variant="h4">Principals</Typography>
+            <Spacer size={10} />
+            <div className="tf__flex-row-center-v">
+              <Typography variant="card-body-text">Estimated</Typography>
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx:176]: use translations please

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+              />
+            </div>
+          </div>
+        </SectionContainer>
+      </SectionContainer>
+      <Spacer size={15} />
+      <SectionContainer title="to">
+        <div className="tf__flex-row-center-v">
+          <SectionContainer>
+            <Typography variant="h4">Principals</Typography>
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx:173]: use translations please

**Review by danieledamiani — CHANGES_REQUESTED** [review]
just the translations

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx]: fixed

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx]: fixed

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx (hunk 3 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx]: fixed

File: packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx (hunk 4 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/removeLiquidity/RemoveLiquidity.tsx]: fixed

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**